app/controllers/usersController.js

```diff
@@ -1,36 +1,27 @@
 const { registerUser } = require('../services/users');
+const { encryptPassword } = require('../helpers/encryption');
+const { isPasswordAlphanumeric, isPasswordLenghtvalid, isEmailValid } = require('../validators/users');
```

Comment:
Create one single user validator function that uses internally these 3 functions.